transport/http/outbound.go

```diff
@@ -88,11 +88,11 @@ func AddHeader(key, value string) OutboundOption {
 	}
 }
 
-// NewOutbound builds an HTTP outbound which sends requests to peers supplied
+// NewOutbound builds an HTTP outbound that sends requests to peers supplied
 // by the given peer.Chooser. The URL template for used for the different
 // peers may be customized using the URLTemplate option.
 //
-// Peer Choosers used with the HTTP outbound MUST yield *hostport.Peer
+// Peer Choosers used with the HTTP outbound MUST yield *httpPeer
```

Comment:
Cool. We should update the documentation to say that instead of referencing the private type.